src/pages/iou/WaypointEditor.js

```diff
@@ -36,6 +36,9 @@ const propTypes = {
         }),
     }),
 
+    /** Recent waypoints array */
+    recentWaypoints: PropTypes.arrayOf(PropTypes.object),
+
```

Comment:
Bad conflicts resolution?

Reply:
It was the source of lint error. I resolved the conflicts

Reply:
@aldo-expensify @narefyev91 

Will you take a look again?